Extensions/Guardrail+Settings.swift

```diff
@@ -9,8 +9,76 @@
 import HealthKit
 
 public extension Guardrail where Value == HKQuantity {
-    static let correctionRange = Guardrail(absoluteBounds: 60...180, recommendedBounds: 70...120, unit: .milligramsPerDeciliter)
+    static let suspendThreshold = Guardrail(absoluteBounds: 67...110, recommendedBounds: 74...80, unit: .milligramsPerDeciliter)
 
+    static func maxSuspendThresholdValue(correctionRangeSchedule: GlucoseRangeSchedule?, preMealTargetRange: DoubleRange?, workoutTargetRange: DoubleRange?, unit: HKUnit) -> HKQuantity {
```

Comment:
[nit] will passing in the unit here work since the `preMealTargetRange` and `workoutTargetRange` are independent of units (and I assume in mg/dL)? 

Consider that units passed in is mmol/L.